db/migrate/20151009033035_add_user_to_assignment_repo.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserToAssignmentRepo < ActiveRecord::Migration
+  def change
+    add_column :assignment_repos, :user_id, :integer
```

Comment:
It looks like you're modifying the DB schema in a way that the code in this pull request won't work correctly without having the migrations run.

If that's the case, users will see errors in production when you deploy this code until the migrations have been run.

As a workaround, you can pull your db migrations out into another pull request that we can deploy first without the functionality you're adding in this code. Then once we run the migrations and merge that pull request you can easily deploy this code without users ever seeing errors.